backend/src/components/user.js

```diff
@@ -203,10 +205,13 @@ function mapUserFacilityObjectForFront(data) {
   return userFacilities
 }
 
-async function getUserFacilities(req, res) {
+async function getUserFacilities(req, res, onlyWithPortalAccess) {
   try {
     let userFacilities = []
```

Comment:
Keep an eye on let/const usage and prefer const wherever possible knowing that const can be mutated just not reassigned.

Reply:
Fixed.